roles/preflight/tasks/main_standalone_containers_certification.yml

```diff
@@ -34,6 +34,18 @@
       loop: "{{ preflight_containers_to_certify }}"
       loop_control:
         loop_var: c
+      when:
+        - not do_container_parallel_test
```

Comment:
I've seen this variable is declared in dci-openshift-app-agent, but not used there. What happens if anyone tries to use this role, just launching it from Ansible and without DCI? Then, we should provide a default value here, and don't know if it's worth to add it to the README too.